app/hotels/src/map/allHotels/HotelSwipeList.js

```diff
@@ -71,26 +77,31 @@ class HotelSwipeList extends React.Component<Props, State> {
     );
   };
 
+  storeRef = (ref: React.ElementRef<typeof Carousel>) => {
+    this.carouselRef = ref;
+  };
+
   render = () => {
-    const { data, selectedIndex, onSnapToItem } = this.props;
+    const { data, onSnapToItem } = this.props;
 
     const child = (
       <BottomSheet openHeight={openHeight} closedHeight={closedHeight}>
         <BottomSheetHandle />
         <View style={styles.sliderWrapper}>
           <Carousel
+            ref={this.storeRef}
             data={data}
             renderItem={this.renderItem}
             sliderWidth={Device.getWideDeviceThreshold()}
             itemWidth={this.getCardItemWidth()}
-            firstItem={selectedIndex}
             inactiveSlideScale={1}
             inactiveSlideOpacity={0.5}
             decelerationRate="fast"
             activeSlideAlignment="start"
             containerCustomStyle={styles.slider}
             removeClippedSubviews={false}
             onSnapToItem={onSnapToItem}
+            useScrollView
```

Comment:
We need to use scroll view with this approach. Default is list view, and list view keeps limited amount in memory. If we snap to an item not in memory, we will se a blank slide only.